fintech-examples/fintech-ui/entry.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+export DOLLAR='$'
```

Comment:
You don't need this line if you use erb, don't you?

Reply:
You're right. Done.